src/v/redpanda/application.cc

```diff
@@ -2047,3 +2043,21 @@ void application::load_feature_table_snapshot() {
       .invoke_on_all([snap](features::feature_table& ft) { snap.apply(ft); })
       .get();
 }
+
+void application::post_start_tasks() {
```

Comment:
Can you comment this new function, e.g., why you would put things here, and what "start" means in the context of "post start".

Reply:
Added a comment stating exactly what "post-start" entails.